common/src/main/java/uk/gov/gchq/palisade/Context.java

```diff
@@ -16,22 +16,40 @@
 
 package uk.gov.gchq.palisade;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 
-public class Justification {
-    private String justification;
+import java.util.HashMap;
+import java.util.Map;
 
-    public Justification() {
+@JsonPropertyOrder(value = {"class", "context"}, alphabetic = true)
+public class Context {
+    private Map<String, Object> context;
```

Comment:
Can we use a different name to prevent confusion with the class name?

Reply:
also the context could just extend HashMap<String, Object>, that way you already have direct access to the getValue, getKeySet methods without first getting the HashMap out of the context object

Reply:
context -> contents